### ISSUE TITLE
Donate button not working

### ISSUE THREAD

**Issue by cocowalla** [oleg-shilo/wixsharp#738]
The "donate" button in your readme file is a [404](https://www.cs-script.net/Donation.html).

**Comment by oleg-shilo** [comment]
Thank you. The url was still pointing to the old hosting address. I missed this one.

Fixed.

**Comment by cocowalla** [comment]
Just used it, can confirm it's working 😄  Thanks for your work on WixSharp - it's really made working with WiX *so* much easier - the documentation, both code docs and the wiki, are also fantastic, as is the support you provide here on GitHub 👍

**Comment by oleg-shilo** [comment]
Thank you, truly appreciate it. 
THE reason for WixSharp existence is the pain and frustration that I have experienced when I tried to work with raw WiX (very long time ago). So I created it to address my own itch. And as with many other tools it just became useful for others.

And even after so many years, after completely stopping dealing with MSI by myself, I still feel like I have to keep it live and kicking so others can breathe easier. :)
